<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AmendThemeDialog</class>
 <widget class="QWidget" name="AmendThemeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Theme Maintance</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/openlp-2.qrc">
    <normaloff>:/icon/openlp.org-icon-32.bmp</normaloff>:/icon/openlp.org-icon-32.bmp</iconset>
  </property>
  <widget class="QDialogButtonBox" name="ThemeButtonBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>500</y>
     <width>156</width>
     <height>26</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="ThemeNameLabel">
      <property name="text">
       <string>Theme Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ThemeNameEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>71</y>
     <width>721</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="LeftSide" native="true">
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>341</width>
         <height>401</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="BackgroundTab">
        <attribute name="title">
         <string>Background</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>321</width>
           <height>351</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="BackgroundLabel">
            <property name="text">
             <string>Background:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QComboBox" name="BackgroundComboBox">
            <item>
             <property name="text">
              <string>Opaque</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Transparent</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="BackgroundTypeLabel">
            <property name="text">
             <string>Background Type:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QComboBox" name="BackgroundTypeComboBox">
            <item>
             <property name="text">
              <string>Solid Color</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gradient</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Image</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="Color1Label">
            <property name="text">
             <string>&lt;Color1&gt;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="2">
           <widget class="QPushButton" name="Color1PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="Color2Label">
            <property name="text">
             <string>&lt;Color2&gt;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="2">
           <widget class="QPushButton" name="Color2PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="ImageLabel">
            <property name="text">
             <string>Image:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QLineEdit" name="ImageLineEdit"/>
          </item>
          <item row="4" column="3">
           <widget class="QPushButton" name="ImagePushButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../images/openlp-2.qrc">
              <normaloff>:/services/service_open.png</normaloff>:/services/service_open.png</iconset>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="GradientLabel">
            <property name="text">
             <string>Gradient :</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2" colspan="2">
           <widget class="QComboBox" name="GradientComboBox">
            <item>
             <property name="text">
              <string>Horizontal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Vertical</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Circular</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="FontMainTab">
        <attribute name="title">
         <string>Font Main</string>
        </attribute>
        <widget class="QGroupBox" name="MainFontGroupBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>307</width>
           <height>119</height>
          </rect>
         </property>
         <property name="title">
          <string>Main Font</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="MainFontlabel">
            <property name="text">
             <string>Font:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QFontComboBox" name="MainFontComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="MainFontColorLabel">
            <property name="text">
             <string>Font Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="MainFontColorPushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="MainFontSize">
            <property name="text">
             <string>Size:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="MainFontSizeLineEdit"/>
          </item>
          <item row="2" column="2">
           <widget class="QSlider" name="MainFontlSlider">
            <property name="value">
             <number>15</number>
            </property>
            <property name="maximum">
             <number>40</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="FooterFontGroupBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>301</width>
           <height>190</height>
          </rect>
         </property>
         <property name="title">
          <string>Display Location</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="FontMainUseDefault">
            <property name="text">
             <string>Use default location</string>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="FontMainXLabel">
              <property name="text">
               <string>X Position:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FontMainXEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="FontMainYLabel">
              <property name="text">
               <string>Y Position:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FontMainYEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="FontMainWidthLabel">
              <property name="text">
               <string>Width</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FontMainWidthEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="FontMainHeightLabel">
              <property name="text">
               <string>Height</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FontMainHeightEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="FontFooterTab">
        <attribute name="title">
         <string>Font Footer</string>
        </attribute>
        <widget class="QGroupBox" name="FooterFontGroupBox_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>301</width>
           <height>190</height>
          </rect>
         </property>
         <property name="title">
          <string>Display Location</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="FontMainUseDefault_2">
            <property name="text">
             <string>Use default location</string>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="FontFooterXLabel">
              <property name="text">
               <string>X Position:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FontFooterXEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="FontFooterYLabel">
              <property name="text">
               <string>Y Position:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FontFooterYEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="FontFooterWidthLabel">
              <property name="text">
               <string>Width</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FontFooterWidthEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="FontFooterHeightLabel">
              <property name="text">
               <string>Height</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FontFooterHeightEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="FooterFontGroupBox_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>307</width>
           <height>119</height>
          </rect>
         </property>
         <property name="title">
          <string>Footer Font</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="FontFooterlabel">
            <property name="text">
             <string>Font:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QFontComboBox" name="FontFooterComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="FontFooterColorLabel">
            <property name="text">
             <string>Font Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="FontFooterColorPushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="FontFooterSizeLabel">
            <property name="text">
             <string>Size:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="FontFooterSizeLineEdit"/>
          </item>
          <item row="2" column="2">
           <widget class="QSlider" name="FontFooterSlider">
            <property name="value">
             <number>15</number>
            </property>
            <property name="maximum">
             <number>40</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="OptionsTab">
        <attribute name="title">
         <string>Alignment</string>
        </attribute>
        <widget class="QGroupBox" name="ShadowGroupBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>301</width>
           <height>80</height>
          </rect>
         </property>
         <property name="title">
          <string>Shadow</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>281</width>
            <height>58</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="ShadowCheckBox">
             <property name="text">
              <string>Use Shadow</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="ShadowColorLabel">
             <property name="text">
              <string>Shadow Color:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="ShadowColorPushButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QGroupBox" name="AlignmentGroupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>321</width>
           <height>161</height>
          </rect>
         </property>
         <property name="title">
          <string>Alignment</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="HorizontalLabel">
            <property name="text">
             <string>Horizontal Align:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="HorizontalComboBox">
            <item>
             <property name="text">
              <string>Left</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Right</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Center</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="VerticalLabel">
            <property name="text">
             <string>Vertical Align:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="VerticalComboBox">
            <item>
             <property name="text">
              <string>Top</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Middle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bottom</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="OutlineGroupBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>301</width>
           <height>80</height>
          </rect>
         </property>
         <property name="title">
          <string>Outline</string>
         </property>
         <widget class="QWidget" name="layoutWidget_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>281</width>
            <height>58</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="OutlineformLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="OutlineCheckBox">
             <property name="text">
              <string>Use Outline</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="OutlineColorLabel">
             <property name="text">
              <string>Outline Color:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="OutlineColorPushButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="RightSide" native="true">
      <widget class="QLabel" name="ThemePreview">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>311</width>
         <height>271</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../images/openlp-2.qrc"/>
 </resources>
 <connections/>
</ui>
